The build does not run benches, and the criterion crate is retired in Fedora.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -38,9 +38,6 @@
 [dependencies.vmm-sys-util]
 version = "0.11.0"
 
-[dev-dependencies.criterion]
-version = "0.3.5"
-
 [features]
 remote_endpoint = []
 test_utilities = []
